plugins/azure-devops/src/components/PullRequestsPage/PullRequestsPage.tsx

```diff
@@ -72,15 +73,23 @@ type PullRequestsPageProps = {
 
 export const PullRequestsPage = (props: PullRequestsPageProps) => {
   const { projectName, pollingInterval, defaultColumnConfigs } = props;
-
-  const { pullRequests, loading, error } = useDashboardPullRequests(
-    projectName,
-    pollingInterval,
+  const config = useApi(configApiRef);
+  const configProjectName = config.getOptionalString(
+    'azureDevOps.repos.projectName',
   );
+  const configPollingInterval = config.getOptionalNumber(
+    'azureDevOps.repos.pollingInterval',
+  );
+  const configDefaultColumnConfig = config.getOptional<
+    PullRequestColumnConfig[]
+  >('azureDevOps.repos.defaultColumnConfigs');
 
-  const [columnConfigs] = useState(
-    defaultColumnConfigs ?? DEFAULT_COLUMN_CONFIGS,
+  const { pullRequests, loading, error } = useDashboardPullRequests(
+    projectName ?? configProjectName,
+    pollingInterval ?? configPollingInterval,
   );
+  const colConfigs = defaultColumnConfigs ?? configDefaultColumnConfig;
+  const [columnConfigs] = useState(colConfigs ?? DEFAULT_COLUMN_CONFIGS);
```

Comment:
It might be clearer to handle defaults in one go.

```suggestion
  const colConfigs = defaultColumnConfigs ?? configDefaultColumnConfig ?? DEFAULT_COLUMN_CONFIGS;
  const [columnConfigs] = useState(colConfigs);
```
or
```suggestion
  const [columnConfigs] = useState(defaultColumnConfigs ?? configDefaultColumnConfig ?? DEFAULT_COLUMN_CONFIGS);
```